codecarbon/external/hardware.py

```diff
@@ -67,6 +67,23 @@ def from_utils(cls, gpu_ids: Optional[List] = None) -> "GPU":
 
 @dataclass
 class CPU(BaseHardware):
-    @property
+    def __init__(self, output_dir: str):
+        self._output_dir = output_dir
```

Comment:
This attribute doesn't seemed used, is that expected?

Reply:
should be used in the subsequent object attribute, which was using the default value before.